scripts/start.js

```diff
@@ -364,7 +369,8 @@ function getUserInfo () {
         resolve({
           userName: username.slice(0, username.indexOf('@')),
           ssoToken: body.access_token,
-          domain: domain
+          domain: domain,
+          userId: userId,
```

Comment:
Please handle the case when userId is not provided by SSO.
The userId is provided by https://gerrit.ovirt.org/#/c/79579/ , so recently in oVirt 4.2 only.
Even if 79579 patch is backported, it will not be in oVirt <= 4.1.4

Please correct me, but I think we shall not increase oVirt API version dependency just because of this.